tests/node_suspension.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Should this test run in the CI?

Reply:
I have commented on this in the description. I want to turn it on in the CI once the #521 is resolved and this test is 100% reliable.